external/avro/src/main/scala/org/apache/spark/sql/v2/avro/AvroScan.scala

```diff
@@ -65,4 +65,8 @@ case class AvroScan(
   }
 
   override def hashCode(): Int = super.hashCode()
+
+  override def getMetaData(): Map[String, String] = {
+    super.metaData ++ Map("Format" -> "avro")
```

Comment:
Could we move all the `Format` metadata into the trait-side?
```
 "Format" -> s"${this.getClass.getSimpleName.replace("Scan", "").toLowerCase(Locale.ROOT)}"
```

Reply:
Also, could we check expain output for Avro V2 scan?

Reply:
@maropu Actually i had tried to test out avro. But i get the following error:
```
"Failed to find data source: avro. Avro is built-in but external data source module since Spark 2.4. Please deploy the application as per the deployment section of "Apache Avro Data Source Guide"
```

Reply:
Ah, I see... @gengliangwang could you help this?

Reply:
@dilipbiswal where do you run the test? I think we have to test it under the external/avro module.

Reply:
@maropu Did you want a explain suite created in the avro module ?

Reply:
Yea if we don't have any other suitable place for adding the test. At least, I think its better to add tests for it somewhere.

Reply:
@maropu OK.. added a test.